package.json

```diff
@@ -21,6 +21,7 @@
     "deploy": "nx deploy",
     "test": "nx test",
     "lint": "nx workspace-lint && nx lint",
+    "clean": "./scripts/clean-generated-files.sh && rm -rf .cache",
```

Comment:
Should also remove:

- `node_modules`?
- `node_modules/.cache`
- `.yarn/cache`?
- `dist`?

Reply:
Moved to own script to allow cleaning even if yarn is broken.

Also, added your ideas.